scripts/package.json

```diff
@@ -3,12 +3,16 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "algolia-setup": "ts-node src/algolia-setup/index.ts"
+    "algolia-setup": "ts-node src/algolia-setup/index.ts",
+    "voucherify-preconfigure": "ts-node src/voucherify-setup/index.ts",
```

Comment:
@marcin-slezak @weronika-kurczyna I get an error when running this against my Voucherify sandbox:

![image](https://github.com/composable-com/composable-ui/assets/13950845/3f2e2198-30f6-489d-96c9-684d9f9e69fe)

Reply:
It looks like your trial has expired. i did not put information in the documentation regarding that adding products is optional, it is not required for proper validation and redemption. So you can use the application without enabling this script - i have updated the documentation. if you need an extended trial on this account let us know, we will probably be able to activate it.